python-lib/design_experiment/constants.py

```diff
@@ -25,6 +25,10 @@ class Parameters(object):
     REACH = "reach"
 
 
-class Group(object):
+class Group(Enum):
     A = "A"
     B = "B"
+
+
+class Column(Enum):
+    AB_GROUP = "__dku_ab_group__"
```

Comment:
imho `dku_ab_group` is better, at first I thought this'd be an internal column, thus the underscore

Reply:
done in https://github.com/dataiku/dss-plugin-ab-testing/pull/1/commits/04bec9de0b9003f6afb292f0275a7d9168664995